ballerina/websocket_service.bal

```diff
@@ -78,15 +98,7 @@ isolated function validateSubscriptionPayload(string text, Engine engine) return
     return {message: "Query not found"};
 }
 
-isolated function broadcast(Engine engine, __Schema schema, Context context, any result,
-                            map<[websocket:Caller, parser:OperationNode]> clientsMap) returns websocket:Error? {
-    foreach var callerDetails in clientsMap {
-        ExecutorVisitor executor = new(engine, schema, context, {}, result);
-        websocket:Caller caller = callerDetails[0];
-        parser:OperationNode operationNode = callerDetails[1];
-        OutputObject outputObject = executor.getExecutorResult(operationNode);
-        if outputObject.hasKey(DATA_FIELD) || outputObject.hasKey(ERRORS_FIELD) {
-            check caller->writeTextMessage(outputObject.toString());
-        }
-    }
-}
+isolated function getSubscriptionResponse(Engine engine, __Schema schema, Context context, string operationName) returns stream<any,error?>|error {
```

Comment:
Isn't this exceeding the maximum character length? The maximum number of characters per line should be under 120.